pythia/bonds.py

```diff
@@ -100,6 +100,9 @@ def neighborhood_distance_sorted(box, positions, neighbors, rmax_guess=2.):
 
 
 def get_neighborhood_angle_matrix(box, positions, neighbors, rmax_guess=2.):
+    """Construct a matrix of pairwise angles between `|r_j - r_i|` and `|r_k - r_i|`
+    for all neighbors j and k(==j) of each particle i. 
```

Comment:
Do you mean `k (k != j)`?

Reply:
actually j can be equal to k so i just deleted it.